src/lib/viewers/controls/color-picker/__tests__/ColorPickerPalette-test.tsx

```diff
@@ -1,12 +1,14 @@
 import React from 'react';
 import { shallow, ShallowWrapper } from 'enzyme';
 import ColorPickerPalette from '../ColorPickerPalette';
+import { AnnotationColor } from '../../../../AnnotationModule';
 
 describe('ColorPickerPalette', () => {
-    const onColorSelect = jest.fn();
+    const colors = Object.values(AnnotationColor);
+    const onSelect = jest.fn();
```

Comment:
This should be inlined as a default value so it's not accidentally reused between tests. Generally, the fewer dependencies each file has, the better.